eureka/S5_lightcurve_fitting/models/Model.py

```diff
@@ -304,13 +328,19 @@ def physeval(self, interp=False, **kwargs):
                     flux *= model.interp(new_time, **kwargs)
                 else:
                     flux *= model.eval(**kwargs)
-        
         return flux, new_time
+    
+    
+    def GP_model_likelihood(self,  fit, **kwargs):
```

Comment:
I don't think this belongs in the gerenal Model.py file - why not just a general function inside the new GPModel.py (no need for it to be a function inside a class).

Reply:
Or it could be in likelihood.py

Reply:
Good point, I'll move it to likelihood.py